package.json

```diff
@@ -86,7 +81,8 @@
     "edgji (https://github.com/edgji)",
     "nlrowe (https://github.com/nlrowe)",
     "npm-to-cdn-bot (by Forbes Lindesay) (https://github.com/npmcdn-to-unpkg-bot)",
-    "polmauri (https://github.com/polmauri)"
+    "polmauri (https://github.com/polmauri)",
+    "Steven Scaffidi (https://github.com/sscaff1)"
```

Comment:
These were originally generated (AFAIK). @eddywashere do you recall what you used to generate this list?